### PR TITLE
Proper image resize

### PR DESCRIPTION
**Pull request by JustHarris**

### CODE REVIEW

File: components/Image/Image.js [hunk 1]
```diff
@@ -4,17 +4,21 @@ import { Image as ImageNative, View } from 'react-native';
 
 import Skeleton from '../Skeleton';
 import styles from './Image.style';
-import resizeWithAkamai from './modules/resizeWithAkamai';
+import resizeWithFastly from './modules/resizeWithFastly';
```
Comment (soyjavi) [inline components/Image/Image.js]: if you don't use `resizeWIthAkamai` you should remove the module.

File: components/Image/modules/resizeWithFastly.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { LAYOUT } from '../../../common';
+
+const AKAMAI_DOMAIN = 'imgs-akamai.mnstatic.com';
+const LOW = { W: 375, H: 736 };
+const REGULAR = { W: 1024, H: 768 };
+const HIGH = { W: 1920, H: 1200 };
+
+export default ({ uri = '', ratio = 1, width = null, height = null }) => {
```
Comment (soyjavi) [inline components/Image/modules/resizeWithFastly.js]: how about some tests?

File: components/Image/Image.js (hunk 1 above)
Comment (JustHarris) [reply components/Image/Image.js]: It is still used in stories and other projects. I have just added it for jeffrey

File: components/Image/modules/resizeWithFastly.js (hunk 2 above)
Comment (JustHarris) [reply components/Image/modules/resizeWithFastly.js]: A test for proper url creation when width and height are requested included

**Comment by veroglez** [comment]
+1

**Pull request merged**